scripts/upload-apk.sh

```diff
@@ -37,13 +34,21 @@ fi
 
 if [[ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]]; then
     for file in app*; do
-          mv ${file} badge-magic-master-${file%%}
+        if [[ ${file} =~ "unsigned" || ${file} =~ "unaligned" ]]; then
```

Comment:
We need unsigned dev apk

Reply:
for what do we need the unsigned apk for?

Reply:
We want it for the dev branch, because we don't sign dev APKs, only master ones. If you remove unsigned dev apk, no APK will be there for dev

Reply:
Umm no. You still will have the debug apk in the Dev branch. Also check the issue description, he specified to remove the unsigned dev apk because it is actually of no use

Reply:
Read the difference about debug and release apk. There'll be no release APK of development branch if you removed unsigned release APK as we don't sign it on development merge

Reply:
https://github.com/fossasia/badge-magic-android/tree/apk

Please point me to the release APK of development branch here which is NOT unsigned, then I'll believe unsigned APK of dev branch is useless

Reply:
> Also check the issue description, he specified to remove the unsigned dev apk because it is actually of no use

Instead of just reading, I actually do some research as well, and don't just say that something is of no use

Reply:
True, there won't be a release apk but we can't install it anyways because it is not signed. So why do you want to keep an apk which is just clogging space. Also **check the issue description**

Reply:
> Please point me to the release APK of development branch here which is NOT unsigned, then I'll believe unsigned APK of dev branch is useless

badge-magic-dev-app-release-unsigned.apk

Reply:
I repeat "which is NOT unsigned", you just proved my point.

> we can't install it anyways because it is not signed

Who said that? Just sign the APK and you can install it.

Reply:
We don't need unsigned and unaligned APK for master branch as there already is a signed APK. Secondly, nothing in APK branch clogs space because it is recreated on evey push

Reply:
@iamareebjamal, I understood it wrong. I understood it 'which is NOT useful'.

Reply:
> Who said that? Just sign the APK and you can install it.

Why don't just sign the apk, I'm not sure if anyone downloads the unsigned app

Reply:
We only sign master APK, but I see no harm in doing that on development as well. If that's easy for you, go ahead

Reply:
Why would it be hard, just need to lift the code out of the if statements.

Reply:
done

Reply:
> Why would it be hard, just need to lift the code out of the if statements.

Because that's not what it is to be done

Reply:
> > Why would it be hard, just need to lift the code out of the if statements.
> 
> Because that's not what it is to be done

why?